stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -131,15 +150,23 @@ llvm::Error interpreterFallback(Operation &op, stablehlo::Scope &scope,
 TranslateFromMLIRRegistration interpretRegistration(
     "interpret", "Interpreter for StableHLO",
     [](ModuleOp module, raw_ostream &os) {
+      auto opsCount = 0;
+      module.walk([&](func::FuncOp funcOp) { opsCount++; });
+
       auto walkResult = module.walk([&](func::FuncOp funcOp) {
-        auto interpreterFallbackFn = [&](Operation &op,
-                                         stablehlo::Scope &scope) {
-          return interpreterFallback(op, scope, funcOp.getSymName());
+        if (opsCount > 1 && funcOp.getSymName() != "main")
+          return WalkResult::advance();
```

Comment:
What happens if there are several functions and there's no `main` function?

Reply:
I've added a check to fail, though no error messages are generated.

Reply:
Let's produce an informative error message to avoid confusion.

Reply:
Done.